pkg/process/runner/submitter.go

```diff
@@ -460,6 +467,26 @@ func (s *CheckSubmitter) shouldDropPayload(check string) bool {
 	return false
 }
 
+func (s *CheckSubmitter) sendHeartbeat() {
```

Comment:
Let's rename this "heartbeat" as its not doing an actual heartbeat for liveness. This is just emitting a process-agent stat.

Reply:
Done